File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        int n=nums1.size();
        int m=nums2.size();
        vector<int> merged;
        for (int i = 0; i < n; i++) {
            merged.push_back(nums1[i]);
        }
        for (int i = 0; i < m; i++) {
            merged.push_back(nums2[i]);
        }
        sort(merged.begin(), merged.end());
        int s=merged.size();
        if (s % 2 != 0)
            return merged[s / 2];
        else
            return (merged[s / 2 - 1] + merged[s / 2]) / 2.0;
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        List<Integer> merged = new ArrayList<>();
        for (int num : nums1) {
            merged.add(num);
        }
        for (int num : nums2) {
            merged.add(num);
        }
        Collections.sort(merged);
        int size = merged.size();
        if (size % 2 != 0) {
            return merged.get(size / 2);
        } else {
            return (merged.get(size / 2 - 1) + merged.get(size / 2)) / 2.0;
        }
    }
}